src/Deserialization/Address.js

```diff
@@ -1,4 +1,4 @@
-/* global require exports */
+/* global require exports BROWSER_RUNTIME */
```

Comment:
`/* global ... */` directives are for some editors to silence "unused variable" warnings. In particular, it works with emacs js2-mode.

Apart from serving their direct purpose, I find them useful because they say something about expectations from runtime.